qulice-findbugs/src/main/java/com/qulice/findbugs/Wrap.java

```diff
@@ -76,6 +75,12 @@ public static void main(final String... args) {
     /**
      * Run it.
      * @param args Arguments
+     * @todo #706  Temporary disable BED_BOGUS_EXCEPTION_DECLARATION check,
+     *  it will be enable after fix
```

Comment:
@dskalenko it should sound more like an exact task to be performed. Let's imagine that brand new joiner gets it and need to have all information required to proceed